datasets/improved_render.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+"""NCS-Render_Shapes.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1hzw2mICnpBPxOYY7-xG-f_EAxB3IYGsQ
+"""
+
+import matplotlib.pyplot as plt
+import torch
+from typing import Callable
+import numpy as np
+import random
+
+#!pip install av
+
+import torchvision
+from matplotlib.pyplot import figure
+from scipy import ndimage
+
+def events_to_frames(frames, polarity: bool = False):
+    if len(frames.shape) == 3:
+        frames = frames.unsqueeze(-1).repeat(1, 1, 1, 3)
+    else:
+        if not polarity:
+            frames = frames.abs().sum(-1)
+        elif polarity:
+            frames = torch.concat(
+                [frames, torch.zeros(*frames.shape[:-1], 1, device=frames.device)],
+                dim=-1,
+            )
+    frames = ((frames / frames.max()) * 255).int()
+    return frames
+
+def rotate_tensor(input, x):
+  rotated_input = torchvision.transforms.functional.rotate(torch.unsqueeze(input, dim = 0), x, expand = True, fill = 0, interpolation = torchvision.transforms.InterpolationMode.NEAREST)
+  return rotated_input[0]
+
+def render_shape(
```

Comment:
Is this not similar to the [`render_shape` function in `render.py`](https://github.com/ncskth/ebv-datasets/blob/main/datasets/render.py#L5)? Could we consider merging them?